### ISSUE TITLE
Geotags infobox "zoom in" doesn't work

### ISSUE THREAD

**Issue by alastair** [MTG/freesound#1038]
JS console shows
```
maps.js:22 Uncaught ReferenceError: GLatLng is not defined
    at setMaxZoomCenter (maps.js:22)
```

**Comment by ffont** [comment]
This is because the function `setMaxZoomCenter` in `maps.js` use old code that accessed Google Maps API V2 (and we are now on V3). In particular, `GLatLng` object should be instantiated differently and the call to `getMaxZoomAtLatLng` should probably be revised as well.

**Comment by ffont** [comment]
Fixed in 8430169bed1aa84015ca3df1593b787644d5ae65
